dotCMS/src/main/java/com/dotcms/contenttype/model/type/Multilinguable.java

```diff
@@ -5,7 +5,9 @@
  * a value in the current language to search with the default language
  * Created by oswaldogallango on 2017-06-19.
  */
-public interface Multilinguable {
+public interface Multilinguable { // Todo: Fallback
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)